models/comment-model.js

```diff
@@ -30,10 +30,10 @@ exports.postCommentQuery = (body, id, username, next) => {
     })
     
     .catch((err) =>{
-        if (err.detail.includes('is not present in table "articles"')) {
+        if (err.detail && err.detail.includes('is not present in table "articles"')) {
             return Promise.reject({status: 404, msg: 'Not found'});
         }
-        else if (err.detail.includes('is not present in table "users"')) {
+        else if (err.detail && err.detail.includes('is not present in table "users"')) {
```

Comment:
A great refactor here as well, stopping a potential typeError popping up if the err.detail was undefined or null